### ISSUE TITLE
Again about observation normalization.

### ISSUE THREAD

**Issue by ZhangHanbo** [sfujim/TD3#11]
In the Supplementary Material, you compared hyperparameters with DDPG from baselines.

In this part, DDPG uses normalized observations. So my question is:

In this paper, which version of codes does the performance of DDPG (not OurDDPG) come from? OpenAI baselines or your implementation? I have some confusion about that because when I run OpenAI's baselines with observation normalization, DDPG does not perform well while your codes perform well. 

I guess that the reason is that the huge replay buffer of DDPG will store all training data which are generated from different normalization parameters.

**Comment by sfujim** [comment]
Oh. Sorry for the confusion. In the paper, DDPG comes from OpenAI baselines. OurDDPG comes from the code provided in this repo. 

DDPG.py in this repo is just to give a better comparison of some of the hyperparameter changes and provide a reference implementation, it is not an exact replication of OpenAI baselines. It is not referenced or used in the paper whatsoever. 

If you have questions about the baselines implementation you should direct it towards them. We ran their code with the default hyperparameters at the time (I don't know if it has since been changed), which are the hyperparameters listed in the supplementary. 

Empirically, I have not found normalization to be beneficial, but this is not something I experimented with greatly. In general, we attempted to simplify the algorithm and preprocessing as much as possible (sticking to defaults, removing unnecessary tricks such as OU noise or normalization). Hope that clears up any confusion!
